share/ipld/nmt_test.go

```diff
@@ -21,10 +21,8 @@ func TestNamespaceFromCID(t *testing.T) {
 	var tests = []struct {
 		randData [][]byte
 	}{
-		{randData: generateRandNamespacedRawData(4, appconsts.NamespaceSize, appconsts.ShareSize)},
-		{randData: generateRandNamespacedRawData(16, 16, appconsts.ShareSize)},
-		{randData: generateRandNamespacedRawData(4, 4, appconsts.ShareSize)},
```

Comment:
changed these tests to only use the standard size. we can switch them back if we want. The new codec has to use slices that exactly 512 bytes, so if we do want to switch them back we just have to adjust for that

Reply:
The exact size is fine, imo. We don't really care about non-canonical sizes